lib/froala-editor-sdk/file.rb

```diff
@@ -77,12 +76,19 @@ def self.save (file, path)
     # +file+:: The file that will be deleted from the server.
     # +path+:: The server path where the file resides.
     # Returns true or false.
-    def self.delete(file = params[:file], path)
+    def self.delete(file = params[:file], options = {})
 
-      file_path = Rails.root.join(path, ::File.basename(file))
-      if ::File.delete(file_path)
-        return true
-      else
+      # Merge options.
+      options = @default_options.merge(options)
+
+      file_path = Rails.root.join(options[:file_access_path], ::File.basename(file))
```

Comment:
I think this change is more for #6.

Reply:
@stefanneculai No, for issue #6 just updating the delete method was enough. Thanks.